Production/test/condorSub/dict_2017B.py

```diff
@@ -0,0 +1,11 @@
+flist = {
+    "scenario": "2017PromptCollisions",
+    "samples": [
+        ['Run2017B-PromptReco-v2.HTMHT'],

```

Comment:
You should also be submitting 2017B-v1 (and eventually 2017C-v2, though none of it is certified yet)...